public/ga.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+  var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+  var ie10Match = navigator.userAgent.match(/MSIE 10/i);
+  var w8Match = navigator.appVersion.match(/Windows NT 6.2/);
+
+  if (fxMatch && Number(fxMatch[1]) < 32) {
```

Comment:
While no doubt important back when this was originally written, this entire block can pretty much go away. Just tap into `navigator.doNotTrack || window.doNotTrack` and directly rely on that.